evennia/utils/tests/test_gametime.py

```diff
@@ -0,0 +1,99 @@
+"""
+Unit tests for the utilities of the evennia.utils.gametime module.
+"""
+
+import time
+import unittest
+from unittest.mock import Mock
+
+from django.conf import settings
+from django.test import TestCase
+
+from evennia.utils import gametime
+
+
+class TestGametime(TestCase):
+    def setUp(self) -> None:
```

Comment:
First time this style of annotation is used in the Evennia code. I still have not gotten used to this (I think it looks pretty messy), but I guess it's fine here.